client/src/context/useKanbanContext.tsx

```diff
@@ -1,17 +1,27 @@
-import { useState, useContext, createContext, FunctionComponent, useEffect, Dispatch, SetStateAction } from 'react';
+import {
+  useState,
+  useContext,
+  createContext,
+  FunctionComponent,
+  useEffect,
+  Dispatch,
+  SetStateAction,
+  useCallback,
+} from 'react';
 import { DraggableLocation, DropResult } from 'react-beautiful-dnd';
 import cloneDeep from 'lodash.clonedeep';
 import { v4 as uuidv4 } from 'uuid';
 import { getBoard, getUserBoards, updateBoard, createBoard } from '../helpers/';
 import { useSnackBar, useAuth } from './';
-import { IKanbanContext, IColumn, ICard, IBoard, NewBoardApiData } from '../interface/';
+import { IKanbanContext, IColumn, ICard, IBoard, INewBoardApiData } from '../interface/';
+import { useHistory } from 'react-router-dom';
 
 export const KanbanContext = createContext<IKanbanContext>({} as IKanbanContext);
 
 export const KanbanProvider: FunctionComponent = ({ children }): JSX.Element => {
   const [activeBoard, setActiveBoard] = useState<IBoard>({
     _id: 'Initial',
-    name: 'Initial',
+    name: '',
```

Comment:
Even though this case is extremely rare, in the event something happens we want to show a title. Maybe `Untitled`?

Reply:
I took this off in order not to need a loading state for the title on the banner bellow the navbar. I change to 'if name==Initial dont display anything'.

Reply:
Could this all be replaced to `useState<IBoard>({} as IBoard);`?

Reply:
I am not sure, I think it's going to trigger null exceptions when functions try to access the columns and so on. Would need some refactoring, but I like the idea.

Reply:
![image](https://user-images.githubusercontent.com/39752144/120906399-628e3880-c616-11eb-937d-e93117c2c924.png)